main.html

```diff
@@ -0,0 +1,365 @@
+      </header>
+      <article class="feature_articles">
+        <summary class="feature_article">
+          <h4>Development</h4>
+          <div>
+            <h5>Web Development</h5>
+            <p>212, 423, 789 students</p>
+          </div>
+          <div>
+            <h5>Python</h5>
+            <p>112, 390, 674 students</p>
+          </div>
+          <div>
+            <h5>Machine Learning</h5>
+            <p>96, 780, 954 students</p>
+          </div>
+        </summary>
+        <summary class="feature_article">
+          <h4>IT and Software</h4>
+          <div>
+            <h5>AWS Certificate</h5>
+            <p>32, 540, 213 students</p>
+          </div>
+          <div>
+            <h5>Ethical Hacking</h5>
+            <p>66, 390, 123 students</p>
+          </div>
+          <div>
+            <h5>Cyber Security</h5>
+            <p>20, 910, 312 students</p>
+          </div>
+        </summary>
+        <summary class="feature_article">
+          <h4>Design</h4>
+          <div>
+            <h5>Photoshop</h5>
+            <p>86, 356, 213 students</p>
+          </div>
+          <div>
+            <h5>Grapics Design</h5>
+            <p>101, 456, 789 students</p>
+          </div>
+          <div>
+            <h5>Drawing</h5>
+            <p>42, 896, 234 students</p>
+          </div>
+        </summary>
+        <summary class="feature_article">
+          <h4>Business</h4>
+          <div>
+            <h5>Financial Analysis</h5>
+            <p>123, 456, 789 students</p>
+          </div>
+          <div>
+            <h5>SQL</h5>
+            <p>36, 390, 213 students</p>
+          </div>
+          <div>
+            <h5>PMP</h5>
+            <p>37, 395, 273 students</p>
+          </div>
+        </summary>
+      </article>
+    </section>
+
+    <!-- Footer Section -->
+    <footer>
+      <div class="partnership">
+        <div class="partnership_heading">
+          <h2>Partner</h2>
+          <div class="partnership_underline"></div>
+        </div>
+        <img
+          src="./assets/-31600822121rk0lmm4sqe.png"
+          alt="airbnb logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/apple-logo.png"
+          alt="apple logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/google-logo.png"
+          alt="google logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/mozilla-logo-11609369622n1zxmrjknf.png"
+          alt="mozilla logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+        <img
+          src="./assets/microsoft-logoo.jpg"
+          alt="microsoft logo"
+          width="200px"
+          height="100px"
+          class="partnership_img"
+        />
+      </div>
```

Comment:
- [ ] kindly remove the margin to match it like the Behance and also remove the open footer tag before the partner section and add it after for the footer part.